src/lib/battery/battery_params_common.c

```diff
@@ -88,3 +88,18 @@ PARAM_DEFINE_FLOAT(BAT_CRIT_THR, 0.07f);
  * @reboot_required true
  */
 PARAM_DEFINE_FLOAT(BAT_EMERGEN_THR, 0.05f);
+
+/**
+ * Battery monitoring source.
+ *
+ * This parameter controls the source of battery data. The value 'Power Module'
+ * means that measurements are expected to come from a power module. If the value is set to
+ * 'External' then the system expects to receive mavlink or UAVCAN battery status messages.
+ *
+ * @min 0
+ * @max 1
+ * @value 0 Power Module
+ * @value 1 External
+ * @group Battery Calibration
+ */
+PARAM_DEFINE_INT32(BAT_SOURCE, 0);
```

Comment:
Why did you move this back? This param was meant to be deprecated.

Reply:
You need a way to turn off the battery_status module. @dagar and I discussed that this is the correct use of this parameter. The BAT%_SOURCE params don't make much sense.

Reply:
Haha, in the current state not, I agree. Unfortunately, I have also some uncommited changes which change it in another way and make `BAT%d_SOURCE` params work. I'll push that soon and we'll have to discuss it all.

Reply:
So you are using the BAT1_SOURCE like I use BAT_SOURCE here.
https://github.com/PX4/Firmware/pull/14490/files#diff-e5c16c5209413f30e964883f74027b51R351
Do you want me to do that? In the UAVCAN case, there could be more than one battery connected and all I need BAT1_SOURCE to do is turn off the battery_status module.